.github/workflows/wheel_linux_aarch64.yml

```diff
@@ -137,7 +137,7 @@ jobs:
           python-version: '3.10'
 
       - name: Install dependencies
-        run: python3.10 -m pip install cibuildwheel~=2.20.0 tomlkit
+        run: python3.10 -m pip install cibuildwheel~=2.21.0 tomlkit
```

Comment:
Could we also remove this pinned version across our wheels recipes? Any concerns with using the [latest_release](https://github.com/pypa/cibuildwheel/releases) version of `cibuildwheel`?

Reply:
I think I'd rather we be explicit about x.y versions, but be unbounded in the bugfix x.y.z version.